lib/superset/dashboard/export.rb

```diff
@@ -23,10 +23,26 @@ def initialize(dashboard_id: , destination_path: )
       end
 
       def perform
+        puts "Starting export for dashboard ID: #{@dashboard_id}"
         create_tmp_dir
+        puts "Temporary directory created at: #{tmp_uniq_dashboard_path}"
+
         save_exported_zip_file
+        puts "Exported zip file saved at: #{zip_file_name}"
+
         unzip_files
-        copy_export_files_to_destination_path if destination_path
+        puts "Files unzipped to: #{tmp_uniq_dashboard_path}"
+
+        copy_export_files_to_destination_path
```

Comment:
@barrynguyen8  ... any particular reason why you removed the condition here?

```
copy_export_files_to_destination_path if destination_path
```

Reply:
Accident, sorry!

Reply:
Reverted https://github.com/rdytech/superset-client/pull/40/commits/8a16a72c90af6b9fccc743505864b77c9021b8fa